lib/isaac_state_manager.go

```diff
@@ -0,0 +1,210 @@
+package sebak
+
+import (
+	"time"
+
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/round"
+)
+
+// ISAACStateManager manages the ISAACState.
+// The most important function `Start()` is called in StartStateManager() function in node_runner.go by goroutine.
+// `transitSignal` is function which is called when the ISAACState is changed.
+// It is used in isaac_state_transit_test.go::TestStateTransitFromTimeoutInitToAccept.
```

Comment:
Please move this documentation on `transitSignal`.
Also, we're having more and more test-only code... That's not a good sign

Reply:
@Geod24 Yes, I agree. But I don't have another good idea for checking state transit u.u.